libs/expo/betterangels/src/lib/screens/Client/Profile/ContactInfo.tsx

```diff
@@ -62,6 +76,81 @@ export default function ContactInfo(props: IProfileSectionProps) {
                 .map(({ label, value }) => (
                   <InfoCol key={label} label={label} value={value} />
                 ))}
+              {client?.clientProfile.phoneNumbers &&
+                client?.clientProfile.phoneNumbers.length > 0 && (
+                  <View>
+                    <View
+                      style={{
+                        alignItems: 'center',
+                        justifyContent: 'space-between',
+                        flexDirection: 'row',
+                        marginBottom: Spacings.sm,
+                      }}
+                    >
+                      <TextRegular size="sm">Phone Number(s)</TextRegular>
+                      <TextBold size="sm">Primary</TextBold>
+                    </View>
+                    {client?.clientProfile.phoneNumbers?.map((phoneNumber) => (
+                      <View
+                        style={{
+                          flexDirection: 'row',
+                          alignItems: 'center',
+                          justifyContent: 'space-between',
+                        }}
+                        key={phoneNumber.id}
+                      >
+                        <TextBold size="sm">{phoneNumber.number}</TextBold>
+                        <PawIcon />
+                      </View>
+                    ))}
+                  </View>
+                )}
+
+              <InfoCol
+                row
+                label="Email"
+                value={client?.clientProfile.user.email}
+              />
+              {client?.clientProfile.socialMediaProfiles &&
+                client?.clientProfile.socialMediaProfiles?.length > 0 && (
```

Comment:
think checking only `client?.clientProfile.socialMediaProfiles?.length` should be enough

It's also good practice to always convert these conditional render statements to a boolean: `!!client?.clientProfile.socialMediaProfiles?.length`

if `client?.clientProfile.socialMediaProfiles` evaluated to `0` then it'd  render `0`

Reply:
changed